# Stamped position/velocity/acceleration for cartesian values and yaw
geometry_msgs/Point pos
geometry_msgs/Vector3 vel
geometry_msgs/Vector3 acc
float32 yaw
float32 yaw_vel
float32 yaw_acc
float32 time